### PR TITLE
Dem/radial multidofs control module

### PR DESCRIPTION
**Pull request by salvalatorre**
Integrating the Radial MultiDOFs module.
Fixed many bugs.
Fixed also the tests and added different features to improve the behavior of the simulations.

### CODE REVIEW

**Comment by salvalatorre** [comment]
Just about to finish, hopefully...

File: applications/DEMApplication/custom_constitutive/DEM_D_Linear_HighStiffness_2D_CL.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    void DEM_D_Linear_HighStiffness_2D::InitializeContact(SphericParticle* const element1, SphericParticle* const element2, const double indentation) {
+        DEM_D_Linear_viscous_Coulomb2D::InitializeContact(element1, element2, indentation);
+        const double kn_augmenter = 1.0;
+        mKn *= kn_augmenter;
+    }
+
+    void DEM_D_Linear_HighStiffness_2D::InitializeContactWithFEM(SphericParticle* const element, Condition* const wall, const double indentation, const double ini_delta) {
+        DEM_D_Linear_viscous_Coulomb2D::InitializeContactWithFEM(element, wall, indentation, ini_delta);
+        const double kn_augmenterFEM = 1.0;
```
Comment (maceligueta) [inline applications/DEMApplication/custom_constitutive/DEM_D_Linear_HighStiffness_2D_CL.cpp]: Are we modifying something at all in this derivate class?

**Comment by salvalatorre** [comment]
@maceligueta
Any news?

File: applications/DEMApplication/tests/test_DEM_schemes.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def GetProblemNameWithPath(self):
         return os.path.join(self.main_path, self.DEM_parameters["problem_name"].GetString())
 
     def CheckValues(self, x_vel, dem_pressure):
-        tol = 1.0e-18
+        tol = 1.0e-10
```
Comment (maceligueta) [inline applications/DEMApplication/tests/test_DEM_schemes.py:41]: increasing tolerance. Why? If it was passing the tests before, the tolerance seems ok, no?

**Pull request merged**